src/services/DiscordRpc.cxx

```diff
@@ -0,0 +1,61 @@
+#include "DiscordRpc.hxx"
+#include "LOG.hxx"
+
+#ifdef USE_DISCORD_RPC
+#include "discord.h"
+
+static std::unique_ptr<discord::Core> core;
+#endif // USE_DISCORD_RPC
+
+void DiscordRpc::init()
+{
+#ifdef USE_DISCORD_RPC
+  discord::Core *newcore{};
+  auto result = discord::Core::Create(980219396125523978, DiscordCreateFlags_Default, &newcore);
```

Comment:
`980219396125523978`
Put it in constexpr with a comment on what it really represents.

Reply:
I think this is the application ID? Please use `982331112061808731` instead